refer to apt.syncthing.net for instructions to avoid discrepancies

nekr0z opened the pull request:
Current Debian guidelines discourage adding system-wide signing
keys for repositories like ours. The reasoning is that in case
such a key is compromised it can be used to sign other
repositories and completely hijack the system, while the new
way of doing things guarantees that the key will only be
considered for our repository, so the attacker must compromise
both the key and the repo to achieve anything, which is more
hassle for the attacker, hence the system is more secure.

Eddy2909 commented:
![left](https://cloud.githubusercontent.com/assets/12877411/9541479/b232f416-4d68-11e5-9166-0f84d566c957.JPG)

nekr0z committed:
don't recommend adding system-wide repository key

Current Debian guidelines discourage adding system-wide signing
keys for repositories like ours. The reasoning is that in case
such a key is compromised it can be used to sign other
repositories and completely hijack the system, while the new
way of doing things guarantees that the key will only be
considered for our repository, so the attacker must compromise
both the key and the repo to achieve anything, which is more
hassle for the attacker, hence the system is more secure.

acolomb commented:
This should probably align with the instructions given at https://apt.syncthing.net/.  Those were changed in a similar manner a few months ago: https://github.com/syncthing/apt-web/pull/23

Maybe just link to the page on the APT repository?  Avoid duplicating information and such inconsistencies as we have now.

nekr0z commented:
> This should probably align with the instructions given at https://apt.syncthing.net/. Those were changed in a similar manner a few months ago: [syncthing/apt-web#23](https://github.com/syncthing/apt-web/pull/23)

You're right. I had no idea we had https://apt.syncthing.net/.

> Maybe just link to the page on the APT repository? Avoid duplicating information and such inconsistencies as we have now.

Makes total sense to me. Will amend.

nekr0z committed:
use apt.syncthing.net as the source of information

imsodin commented on content/downloads.md:
````diff
@@ -39,55 +39,6 @@ running on an oddball system such as a NAS, please consult your vendor.
-```
-
-If you insist, you can also use the above URLs with http instead of https.
+Packages and installation instructions are available at [apt.syncthing.net](https://apt.syncthnig.net).
````
```suggestion
Packages and installation instructions are available at [apt.syncthing.net](https://apt.syncthing.net).
```

nekr0z committed:
fix link

Co-authored-by: Simon Frei <freisim93@gmail.com>

nekr0z commented:
@imsodin thank you for catching that!
